Plugins/DataSink/MQTT/MQTTConfig.cs

```diff
@@ -8,7 +8,7 @@ public class MQTTConfig : ConfigLoader<MQTTConfig>, IConfigSaver
         /// <summary>
         /// IP of MQTT broker to connect to
         /// </summary>
-        public string BrokerIP { get; set; } = "127.0.0.1";
+        public string BrokerIP { get; set; } = "localhost";

```

Comment:
`127.0.0.1` will be always available. You can modify the `localhost` entry in the host file to point to somewhere else.

Reply:
on the other hand localhost is protocol independent (works for IPv4 and IPv6). and there are a few occasions of localhost already in the code.
tell me which one you prefer and I will change it everywhere :)